packs/azure/actions/lib/formatters.py

```diff
@@ -0,0 +1,16 @@
+__all__ = [
+    'to_node_dict'
+]
+
+
+def to_node_dict(node):
```

Comment:
We can create /opt/stackstorm/libs/ and throw these common code in there. When we create virtualenvs for actions or sensors, we can create symlinks or just add this path to PYTHONPATH.